### PR TITLE
Allow disabling felix

### PR DESCRIPTION
**Pull request by jaxxstorm**
helps with #1518

### CODE REVIEW

File: calico_node/filesystem/etc/rc.local [hunk 1]
```diff
@@ -27,8 +27,10 @@ mkdir /etc/service/enabled
 # dirs contained in Docker image. This is due to bug in Docker with graphdriver
 # overlay on CentOS 7.X kernels (https://github.com/docker/docker/issues/15314)
 
-# Felix is always enabled
-cp -a /etc/service/available/felix /etc/service/enabled/
+# In certain unique situation we want to disable felix
```
Comment (fasaxc) [inline calico_node/filesystem/etc/rc.local]: How about adding an example:  `# Allow Felix to be disabled, for example, if the user is running Felix outside the container.`

Comment (jaxxstorm) [inline calico_node/filesystem/etc/rc.local]: Good idea, fixed!

**Comment by fasaxc** [comment]
@heschlie Can this go in, or are frozen for the release?

**Comment by jaxxstorm** [comment]
ping? @heschlie

**Pull request merged**